test/unit/karma.conf.js

```diff
@@ -32,7 +32,7 @@ module.exports = function (config) {
         },
         files: [
             'kbase-extension/static/narrative_paths.js',
-            {pattern: 'test/unit/spec/**/*.js', included: false},
+            {pattern: 'test/unit/spec/**.js', included: false},
```

Comment:
Is this still supposed to be different?

Reply:
Nope, that's what causing probably only a small handful of tests to run. :)